routes/auth.routes.js

```diff
@@ -0,0 +1,30 @@
+const express = require("express");
+const router = express.Router();
+const authController = require("../controllers/auth.controller");
+const passport = require("passport");
+
+const auth = (req, res, next) => {
```

Comment:
To masz w 2 miejscach (i pewnie przy kolejnych endpointach kopiował byś to dalej) wyrzućmy to do osobnego pliku i importujmy to tu i w contacts.